### ISSUE TITLE
Butter project independence

### ISSUE THREAD

**Issue by streger** [butterproject/butter-desktop#677]
There is a guide for how to maintain a open-source project here:
https://github.com/nayafia/lemonade-stand

Please do post your ideas how to make Butter sustainable and reasons for ours work.

**Comment by WildYorkies** [comment]
@streger Are you part of the Butter organization? I don't think the lack of development is a money issue. If it is, they could easily get a big patreon crowd fund. But I really doubt that's the issue.

**Comment by streger** [comment]
Yes I'm from the start. Everything on the planet is a money (time) issue.
As much as I can see from your post is that you think that we could somehow brake the the ideas of the current state of 'media' licences and share that with the world so we somehow bypass it and get support form big crowd. That's never going to happen.

Butter project is about making a software product that is pointed at general public for easy implementation of almost any kind of video (or not) web streaming application that suits anyone's imagined idea what streaming application should represent.
It doesn't include the idea of having an application that streams movies and TV shows from america.

**Comment by WildYorkies** [comment]
@streger Are you only asking for advice on the core butter project and **not** the butter-desktop application?

I can only speak for the butter-desktop application, but people have been asking if this project has been abandoned for at least 6 months now. Nobody knows what's going on with it. A lack of base level communication shouldn't be a funding issue. To me, that reads as a lack of interest from the developers.

To understand what is needed, the communication has to be there. Otherwise, I don't think the users know what to offer.

Let me know if that makes sense or not.

**Comment by vankasteelj** [comment]
We kinda stopped communicating because we have nothing to communicate. 

Here's my stand:
The project isn't dead. At least I don't think it is. But it has been hit pretty hard by the lack of developers. We don't want money, we want passion. And pouring all our free time in something that doesn't seem to get people together, somewhat used our passion little by little. I've started a few side projects on my own. One day that work could be integrated into Butter, so that's a plus, but it isn't easy on the moral to give passion to Butter when your passion is actually being appreciated elsewhere. 

Also, video games and TV Series. :)
